### ISSUE TITLE
Unable to build project

### ISSUE THREAD

**Issue by NicoCG** [Yucked/Valerie#47]
**Describe the bug**
Unable to build project. 
Did not change anything.
I am running Visual Studio 2017 fully updated with all required packages installed.

**To Reproduce**
1. Click on 'Build'
2. See attachment for error's

**Screenshots**
Errors:
![errors](https://user-images.githubusercontent.com/33652180/41103058-8a3ff8ee-6a68-11e8-8d37-beec8dba331d.PNG)
Packages:
![packages](https://user-images.githubusercontent.com/33652180/41103193-d012642e-6a68-11e8-9370-2d92411b369a.PNG)

**Desktop (please complete the following information):**
 - OS: [ Windows 10]

**Comment by Yucked** [comment]
Seems like it's unable to install Discord.Net or the latest one at least. 
Open a console window where `Valerie.csproj` is located at and type `dotnet restore`. That should be able to restore all the packages from `nuget.config`.
If that doesn't work try ->
In Visual Studio, click on `Tools` -> `Nuget Package Manager` -> `Package Manager Settings` -> `Package Resources` 
Then click on the `+` sign and add a new package source. Put **Name** as `Discord.Net`  and **Source** is: `https://www.myget.org/F/discord-net/api/v3/index.json`
Do the same for RavenDB as well. Source for RavenDB is: `https://www.myget.org/F/ravendb/api/v3/index.json`

Once that is done open `Package Manager Console` and type `dotnet restore`

**Comment by NicoCG** [comment]
Thanks that helped a lot adding this 

> In Visual Studio, click on Tools -> Nuget Package Manager -> Package Manager Settings -> Package Resources
> Then click on the + sign and add a new package source. Put Name as Discord.Net and Source is: https://www.myget.org/F/discord-net/api/v3/index.json
> Do the same for RavenDB as well. Source for RavenDB is: https://www.myget.org/F/ravendb/api/v3/index.json

For some odd reason nuget could not find RavenDB version 4.0.4-nightly-20180531-0447 so I had to install 4.0.4-patch-40038

All is working now thanks for the cool bot and if you ever thinking of new features adding some form of translator would be really cool! not sure if that's possible.

**Comment by Yucked** [comment]
You can install the latest RavenDB package from here: https://www.myget.org/feed/ravendb/package/nuget/RavenDB.Client/4.0.4-nightly-20180607-0802

And the translator feature is planned. Just trying to find a decent API that doesn't require a key or oauth stuff.

**Comment by NicoCG** [comment]
Thanks, That worked!

I cannot wait for future releases its probably the most decent bot out there.

**Comment by Yucked** [comment]
Aw thank you.
I'll be closing this issue now. If you encounter any further issues feel free to open a new issue.
